website/cue/reference/components/sinks/base/nats.cue

```diff
@@ -792,6 +792,6 @@ base: components: sinks: nats: configuration: {
 			[nats_url]: https://docs.nats.io/using-nats/developer/connecting#nats-url
 			"""
 		required: true
-		type: string: examples: ["nats://demo.nats.io", "nats://127.0.0.1:4242"]
+		type: string: examples: ["nats://demo.nats.io", "nats://127.0.0.1:4242", "nats://localhost:4222,nats://localhost:5222,nats://localhost:6222"]
```

Comment:
Are these separate examples or one entire example? I wanted to verify since the newest additions are all under the same quotation mark.

```suggestion
		type: string: examples: ["nats://demo.nats.io", "nats://127.0.0.1:4242", "nats://localhost:4222, nats://localhost:5222, nats://localhost:6222"]
```

Reply:
This is one entire example, we can now pass several urls as one string, urls should be separated by comma. No spaces is a standard for setting several urls for nats, vector nats source settings work just the same https://github.com/vectordotdev/vector/blob/c3d91af5288cc4d10806f7724bf7ba0401ba2b7e/website/cue/reference/components/sources/base/nats.cue#L662